addons/web_editor/static/src/scss/wysiwyg.scss

```diff
@@ -264,6 +264,8 @@ body .modal {
             .o_existing_attachment_cell {
                 cursor: pointer;
                 margin: 1px;
+                overflow: hidden;
+                text-overflow: ellipsis;
```

Comment:
- I see the bug is also there in at least 14.0 (maybe even before?) so it should be fixed there :+1: 
- Those rules would be better applied only to the deepest element which contains the text which might overflow. You don't want `overflow: hidden` to apply to large containers for no reason.
- There is a mixin (o-text-overflow ?) which does that, maybe even a class I am not sure. At least use the mixin :+1:

Reply:
Thanks for the review. I'll look into this and create a new PR for 14.0 or before for you to review then :-)

Reply:
@qsm-odoo I was able to track it back to at least saas-12.3. I suspect the problem would be there in 12.0 too but there the documents tab looks like this:
![image](https://user-images.githubusercontent.com/24205914/116212566-3d6bf780-a745-11eb-9eda-748344b9ea5d.png)
Should I apply the (improved) fix to 12.3 for the time being?